mathics/builtin/datentime.py

```diff
@@ -987,7 +986,7 @@ class DateStringFormat(Predefined):
 
     <dl>
       <dt>'$DateStringFormat'
-      <dd>gives the format used for dates generated by 'DateString'.
+      <dd>get the format used for dates generated by 'DateString'.
```

Comment:
I had the feeling that in the docstring we used the style "[Expression] does/returns/gives ..." using a third person in the simple present tense, while the `summary_text` was in the imperative form, wasn't it?

Reply:
This should have just been on the summary text. I'll change this. 

In the summary text it is 3rd person active. "get" or "give" would work but for summary text "get" seems to be what is used most.